src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -1,28 +1,24 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
-import { Stack, TextField } from '@mui/material';
+import { Stack } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
-import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
```

Comment:
does it mean that `MuiDatePicker` doesn't work with _date-fns_ anymore ? If it is so, we could consider one day,  migrating completely to _dayjs_ to avoid two identical dependencies :slightly_smiling_face:

Reply:
Unfortunately, Only AdapterDayjs, AdapterLuxon and AdapterMoment are currently compatible with UTC dates and timezones.

Reply:
Ok, thank you for the answer ! we can take note of it and pass to full _dayjs_ one day :+1: